bpf/lib/lb.h

```diff
@@ -521,7 +522,7 @@ static __always_inline void
 __lb6_update_affinity(const struct lb6_service *svc, bool netns_cookie,
 		      union lb6_affinity_client_id *id, __u32 backend_id)
 {
-	__u32 now = bpf_ktime_get_sec();
+	__u32 now = bpf_mono_now();
```

Comment:
Does the usage of `now` below need to be converted from mono -> sec?

Same comment for v6 below.

Reply:
No, why? The timeout is converted to `bpf_sec_to_mono(svc->affinity_timeout)`, so we need `bpf_mono_now()` to  store current value.

Reply:
This function stores `jiffies >> BPF_MONO_SCALAR` into `svc->affinity_timeout`, not seconds (there's no `KERNEL_HZ` handling).

The code later applies `bpf_sec_to_mono(svc->affinity_timeout)` when reading the value out, ie `bpf_sec_to_jiffies(svc->affinity_timeout) >> BPF_MONO_SCALAR`. If we're storing jiffies, then why do we then multiply that by `KERNEL_HZ` and further right-shift the value? Why convert from seconds -> jiffies if the value is already jiffies?

Reply:
Looks like I misread, this is actually storing into `last_used` and the logic later that does math against `last_used`, `affinity_timeout` and `now` are all correctly converted over to jiffies in this patch. So LGTM.